python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -315,13 +316,15 @@ class NearestNeighbors(Base,
         self.algorithm = algorithm
         self.algo_params = algo_params
         self.knn_index = <uintptr_t> 0
+        self.n_jobs = n_jobs
```

Comment:
do we need to store it?

Reply:
Oops!  Forgot to remove this;  thanks, Dante!